python/TestHarness/TestHarness.py

```diff
@@ -544,6 +544,7 @@ def parseCLArgs(self, argv):
         parser.add_argument('--devel', action='store_const', dest='method', const='devel', help='test the app_name-devel binary')
         parser.add_argument('--oprof', action='store_const', dest='method', const='oprof', help='test the app_name-oprof binary')
         parser.add_argument('--pro', action='store_const', dest='method', const='pro', help='test the app_name-pro binary')
+        parser.add_argument('--run', type=str, default='', dest='run', help='only run tests of the specified of tag(s)')
```

Comment:
Looking at this now in the code, I don't think it's really necessary. This is almost exactly the same functionality as the group functionality. So now we have two ways to specify test subsets and two ways to run them. If this is all you need, let's use group since it's already in there.

Reply:
Sure - I'll switch it.

Reply:
The problem is that all groups are run by default, and we don't want benchmarks to run by default.  We would need a way to have default-disabled groups that only run if explicitly listed on the cli.